src/lang/english.txt

```diff
@@ -2033,6 +2033,7 @@ STR_NETWORK_SERVER_LIST_SERVER_VERSION                          :{SILVER}Server
 STR_NETWORK_SERVER_LIST_SERVER_ADDRESS                          :{SILVER}Server address: {WHITE}{RAW_STRING}
 STR_NETWORK_SERVER_LIST_START_DATE                              :{SILVER}Start date: {WHITE}{DATE_SHORT}
 STR_NETWORK_SERVER_LIST_CURRENT_DATE                            :{SILVER}Current date: {WHITE}{DATE_SHORT}
+STR_NETWORK_SERVER_LIST_GAMESCRIPT                              :{SILVER}Gamescript: {WHITE}{RAW_STRING} (v{NUM})
```

Comment:
"Gamescript" here could also be rewritten as "Game Script" (the latter wording is used elsewhere).